src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -46,20 +51,20 @@ public void initMocks() {
 
   @Test
   public void testGetCaseByCaseId_withCaseDetails() throws Exception {
-    doTestGetCaseByCaseId(true);
+    doTestGetCaseByCaseId(CaseType.H, true);
   }
 
   @Test
   public void testGetCaseByCaseId_withNoCaseDetails() throws Exception {
-    doTestGetCaseByCaseId(false);
+    doTestGetCaseByCaseId(CaseType.C, false);
```

Comment:
testGetCaseByCaseId_withCaseDetails and testGetCaseByCaseId_withNoCaseDetails use two different args when calling doTestGetCaseByCaseId - should they not vary the criteria more subtly or are more variations of this test required?

Reply:
More permutations added. 

I was taking a shortcut, and slipping testing of the household/communal filtering into the events on/off tests.